jenkins/add-docker-build-job.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+set -e
+
+if [ "$#" -lt 7 ]; then
+    echo ""
+    echo "Usage: ./add-docker-build-job.sh <jenkins url>"
+    echo "                                 <jenkins user>"
+    echo "                                 <jenkins password>"
+    echo "                                 <git url>"
+    echo "                                 <container registry url>"
+    echo "                                 <container registry user name>"
+    echo "                                 <container registry password>"
+    echo "                                 [opt: credentials id]"
+    echo "                                 [opt: credentials description]"
+    echo "                                 [opt: job short name]"
+    echo "                                 [opt: job display name]"
+    echo "                                 [opt: job description]"
+    echo "                                 [opt: container tag prefix]"
+    exit 1
+fi
+jenkins_url=${1}
+jenkins_user=${2}
+jenkins_password=${3}
+git_url=${4}
+cr_url=${5}
+cr_user=${6}
+cr_password=${7}
+cr_credentials_id=${8}
+cr_credentials_description=${9}
+job_name=${10}
+job_display_name=${11}
+job_description=${12}
+container_tag_prefix=${13}
+
+if [ -z "$cr_credentials_id" ]
+then
+    cr_credentials_id="docker_credentials"
+fi
+if [ -z "$cr_credentials_description" ]
+then
+    cr_credentials_description="Docker Container Registry Credentials"
+fi
+if [ -z "$job_name" ]
+then
+    job_name="basic-docker-build"
+fi
+if [ -z "$job_display_name" ]
+then
+    job_display_name="Basic Docker Build"
+fi
+if [ -z "$job_description" ]
+then
+    job_description="A basic pipeline that builds a Docker container. The job expects a dockerfile inside the git repository"
```

Comment:
Does the dockerfile have to be at the root of the repo? If so I would say "The job expects a dockerfile at the root of the git repository"